strider/trapi.py

```diff
@@ -166,6 +167,51 @@ def is_ancestor(a, b):
     return [category for category, drop in zip(categories, has_descendant) if not drop]
 
 
+def filter_information_content(
+    message: Message,
+    curie_map: dict,
+    logger: logging.Logger = logging.getLogger(),
+    information_content_threshold: int = settings.information_content_threshold,
+) -> None:
+    """Filter all nodes based on information content."""
+    keep_edges = []
+    new_results = Results.parse_obj([])
+    for result in message.results or []:
+        keep = True
+        for node_bindings in result.node_bindings.values():
+            for node_binding in node_bindings:
+                curie = curie_map.get(node_binding.id)
+                if (
+                    curie is not None
```

Comment:
Is there a legitimate reason this could happen? I get it as a safety, but do we need it? How about the opposite as a safety?

Reply:
I think there's a good probability that this can happen occasionally. The `curie_map` is all of the nodes that the node normalizer knows about in the given message, but there's a chance that a node in some kp's response isn't known by node norm.